nectar/.geth_datadir/UTC--2020-06-24T07-35-41.322460345Z--88552c5d1a30ac899e3d17c63754f3f257e42663

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
I would to know what this file is used for, seem redundant.

Reply:
@D4nte, @da-kami, @luckysori any insights?

Reply:
I am running tests without it and see what happens ;)

Reply:
I confirm it's needed for the tests :)
```
test swap::tests::execute_alice_hbit_herc20_swap ... FAILED

failures:

---- ethereum::geth::test::get_chain_id stdout ----
thread 'ethereum::geth::test::get_chain_id' panicked at 'called `Result::unwrap()` on an `Err` value: No such file or directory (os error 2)', src/ethereum/geth.rs:228:64

---- ethereum::wallet::tests::can_deploy_htlc stdout ----
thread 'ethereum::wallet::tests::can_deploy_htlc' panicked at 'called `Result::unwrap()` on an `Err` value: No such file or directory (os error 2)', src/ethereum/wallet.rs:567:55

---- ethereum::wallet::tests::ether_balance stdout ----
thread 'ethereum::wallet::tests::ether_balance' panicked at 'called `Result::unwrap()` on an `Err` value: No such file or directory (os error 2)', src/ethereum/wallet.rs:456:55

---- ethereum::wallet::tests::gas_limit stdout ----
thread 'ethereum::wallet::tests::gas_limit' panicked at 'called `Result::unwrap()` on an `Err` value: No such file or directory (os error 2)', src/ethereum/wallet.rs:488:55

---- ethereum::wallet::tests::gas_price stdout ----
thread 'ethereum::wallet::tests::gas_price' panicked at 'called `Result::unwrap()` on an `Err` value: No such file or directory (os error 2)', src/ethereum/wallet.rs:472:55

---- ethereum::wallet::tests::transfer_dai stdout ----
thread 'ethereum::wallet::tests::transfer_dai' panicked at 'called `Result::unwrap()` on an `Err` value: No such file or directory (os error 2)', src/ethereum/wallet.rs:512:55

---- swap::tests::execute_alice_hbit_herc20_swap stdout ----
Error: No such file or directory (os error 2)
thread 'swap::tests::execute_alice_hbit_herc20_swap' panicked at 'assertion failed: `(left == right)`
  left: `1`,
 right: `0`: the test returned a termination value with a non-zero status code (1) which indicates a failure', /Users/froyer/.rustup/toolchains/1.46.0-x86_64-apple-darwin/lib/rustlib/src/rust/src/libstd/macros.rs:16:9


failures:
    ethereum::geth::test::get_chain_id
    ethereum::wallet::tests::can_deploy_htlc
    ethereum::wallet::tests::ether_balance
    ethereum::wallet::tests::gas_limit
    ethereum::wallet::tests::gas_price
    ethereum::wallet::tests::transfer_dai
    swap::tests::execute_alice_hbit_herc20_swap

test result: FAILED. 140 passed; 7 failed; 7 ignored; 0 measured; 0 filtered out

error: test failed, to rerun pass '--bin nectar'
```